src/operator/OrgOverlay.tsx

```diff
@@ -0,0 +1,242 @@
+import React, {FC, useContext} from 'react'
+import {useHistory, useParams} from 'react-router-dom'
+
+import {
+  ButtonBase,
+  ButtonShape,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  Form,
+  Gradients,
+  Grid,
+  InputType,
+  LinkButton,
+  LinkTarget,
+  Overlay,
+  RemoteDataState,
+  SpinnerContainer,
+  TechnoSpinner,
+} from '@influxdata/clockface'
+
+import {OverlayContext} from 'src/operator/context/overlay'
+import LimitsInput from 'src/operator/LimitsInput'
+import {fromDisplayLimits} from 'src/operator/utils'
+
+const OrgOverlay: FC = () => {
+  const {
+    limits,
+    limitsStatus,
+    handleUpdateLimits,
+    organization,
+    orgStatus,
+    setLimits,
+    updateLimitStatus,
+  } = useContext(OverlayContext)
+
+  const {orgID} = useParams()
+  const history = useHistory()
+
+  const setInputs = limits => {
+    setLimits(limits)
+  }
+
+  const updateLimits = async () => {
+    try {
+      const backendLimits = fromDisplayLimits(limits)
+      await handleUpdateLimits(backendLimits)
+      history.goBack()
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  // TODO(ariel): fix the background issue by this being on a separate page
+  return (
+    <Overlay
+      visible={true}
+      renderMaskElement={() => (
+        <Overlay.Mask gradient={Gradients.PowerStone} style={{opacity: 0.5}} />
+      )}
+      testID="org-overlay"
+      transitionDuration={0}
+    >
+      <Overlay.Container maxWidth={1000}>
+        <Overlay.Header
+          title={orgID}
+          style={{color: '#FFFFFF'}}
+          onDismiss={() => history.goBack()}
+        />
+        <SpinnerContainer
+          loading={orgStatus}
+          spinnerComponent={<TechnoSpinner diameterPixels={100} />}
+        >
+          <Overlay.Body>
+            <Grid>
```

Comment:
The Quartz version of this component has an `Alert` being shown in case of a request error. How is that handled here?

Reply:
This is being handled by a global alert notification that we're dispatching in the context provider